src/components/page-configuration/delete-application-form.js

```diff
@@ -53,42 +61,53 @@ export class DeleteApplicationForm extends React.Component {
             </p>
             <div className="o-action-bar">
               <Button color="danger" onClick={this.handleClick}>
-                Delete app
+                Delete application
               </Button>
             </div>
           </div>
           {this.state.visibleScrim && (
             <Scrim onClose={this.handleClick} isDismissable className="scrim">
               <div className="delete-app">
                 <div className="header-actions">
-                  <h6>Delete application</h6>
+                  <Typography group="ui" variant="accordion_header">
+                    Delete {this.props.appName}
+                  </Typography>
                   <Button variant="ghost_icon" onClick={this.handleClick}>
                     <Icon data={clear} />
                   </Button>
                 </div>
                 <div className="accordion__content">
-                  <p className="body_short">
-                    This action can not be undone. You will permanently remove
-                    application-name from Radix including all its environment.
-                  </p>
-                  <p className="body_short">
+                  <Typography>This action can not be undone.</Typography>
```

Comment:
I think it would be better to style "This action can not be undone" as a warning Alert with the icon you have put to the right of the Delete button.